lab_2a_handshake/ClientProtocol.py

```diff
@@ -25,6 +25,13 @@ def connection_made(self, transport):
 			print("Client Side: Connection Made...")
 		self.transport = transport
 
+	def timeout_checker(self):
+		if self.state == "SYN_ACK_State_1":
+			print("Time-out. Close Connection.")
```

Comment:
Please follow the print formation: "Client Side: balabala"
(You can see other print example in this file)